### PR TITLE
PZA-40 Absolute pathing implemented

### PR DESCRIPTION
**Pull request by Prashishgt**
Absolute pathing has been implemented according to requirment, although there are few things that need to addressed.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
-import Navbar from 'Layouts/nav/Navbar';
-import HomeRoutes from 'Routes/HomeRoutes';
-import Footer from 'Layouts/footer/Footer';
-import './scss/main.scss';
-import UnderConstructionPage from './pages/UnderConstruction/UnderConstructionPage';
+import Navbar from 'layouts/nav/Navbar';
+import HomeRoutes from 'routes/HomeRoutes';
+import Footer from 'layouts/footer/Footer';
+import 'scss/main.scss';
```
Comment (sarojghalan) [inline src/App.tsx:5]: scss import should be at last.

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,17 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (dipeshkumarsah98) [inline src/App.tsx:1]: import styles like bootstrap and scss in the last

File: src/Layouts/Athleisure/Athleisure.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { displaypic, athleisureback, athleisurefront } from 'assets/Images';
```
Comment (dipeshkumarsah98) [inline src/Layouts/Athleisure/Athleisure.tsx:1]: same as above, import components first then assets like image or css

File: src/Layouts/footer/Footer.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import { ArrowRightAltIcon } from 'assets/icons';
```
Comment (dipeshkumarsah98) [inline src/Layouts/footer/Footer.tsx:3]: same as above import assests in last

File: src/components/Card/ProductCard.tsx [hunk 5]
```diff
@@ -1,9 +1,8 @@
-import '../../scss/main.scss';
 import React, { useState } from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/Card/ProductCard.tsx:1]: do we need react in every components ?

File: src/components/Card/ProductCard.tsx [hunk 6]
```diff
@@ -1,9 +1,8 @@
-import '../../scss/main.scss';
 import React, { useState } from 'react';
 import Drawer from '@mui/material/Drawer';
```
Comment (dipeshkumarsah98) [inline src/components/Card/ProductCard.tsx:2]: follow this order while importing.
- React core lib
- components
- external components
- assets/image
- styles

File: src/components/CartTable/CartTable.tsx [hunk 7]
```diff
@@ -1,5 +1,5 @@
+import { CloseIcon } from 'assets/icons';
 import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/CartTable/CartTable.tsx:2]: same as above, 
if we don't need react in pure component then remove it!! 
at first check whether it is working without react or not.

File: src/components/CartTable/CartTable.tsx [hunk 8]
```diff
@@ -1,5 +1,5 @@
+import { CloseIcon } from 'assets/icons';
```
Comment (dipeshkumarsah98) [inline src/components/CartTable/CartTable.tsx:1]: follow the orders of import i mentioned above

File: src/components/Checkout/CheckoutInformation.tsx [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/Checkout/CheckoutInformation.tsx:1]: same, do we need to import react ?
if not then remove from all the files.

File: src/components/Checkout/CheckoutLeftItem.tsx [hunk 10]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Breadcrumbs } from '@mui/material';
```
Comment (dipeshkumarsah98) [inline src/components/Checkout/CheckoutLeftItem.tsx:2]: here as well. follow the import order

File: src/components/Checkout/CheckoutLeftItem.tsx [hunk 11]
```diff
@@ -1,7 +1,7 @@
...
-import NavigateNextIcon from '@mui/icons-material/NavigateNext';
 import { Link, Outlet } from 'react-router-dom';
+import { NavigateNextIcon } from 'assets/icons';
 
 function CheckoutLeftItem() {
   // code for breadcrumbs
```
Comment (dipeshkumarsah98) [inline src/components/Checkout/CheckoutLeftItem.tsx:7]: check if we can make breadcrumbs dynamic ? if yes, then let's do it.

File: src/components/Checkout/PaymentComponent.tsx [hunk 12]
```diff
@@ -1,7 +1,7 @@
 // Import required modules
 import React from 'react';
-import InputRhude from 'components/ui/input/InputRhude'; // InputRhude component
-import Button from 'components/button/Button'; // Button component
+import { InputRhude } from 'components/ui/input'; // InputRhude component
```
Comment (dipeshkumarsah98) [inline src/components/Checkout/PaymentComponent.tsx:3]: remove unnecessary comments and why we are importing react ?

File: src/components/CountDown/CountDown.tsx [hunk 13]
```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from 'react';
-import ArrowDropDownCircleIcon from '@mui/icons-material/ArrowDropDownCircle';
+import { ArrowDropDownCircleIcon } from 'assets/icons';
```
Comment (dipeshkumarsah98) [inline src/components/CountDown/CountDown.tsx:2]: again follow import standard import order.

File: src/components/modal/ModalComponent.tsx [hunk 14]
```diff
@@ -1,10 +1,10 @@
 import React, { useState, useEffect } from 'react';
+import { CloseIcon } from 'assets/icons';
 import Box from '@mui/material/Box';
 import Modal from '@mui/material/Modal';
-import CloseIcon from '@mui/icons-material/Close';
 import { IconButton } from '@mui/material';
 import { CountryType, countries } from 'data/country';
-import Message from 'Provider/Message';
+import Message from 'provider/Message';
```
Comment (dipeshkumarsah98) [inline src/components/modal/ModalComponent.tsx:7]: again follow the standard import order

File: src/pages/Checkout.tsx [hunk 15]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/pages/Checkout.tsx:1]: again react ?

**Review by dipeshkumarsah98 — CHANGES_REQUESTED** [review]
Nice work ! Some of the changes are required which i have commented out. And server is still not running so solve that issue as well. I have attached the screenshot of the error. When you complete the changes,  you can ask for re-review. thank you. Really appreciate your work :+1: .
image

**Review by sarojghalan — COMMENTED** [review]
Well done .

File: src/Count.tsx [hunk 16]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, ChangeEvent } from 'react';
+import React, { useState } from 'react';
```
Comment (dipeshkumarsah98) [inline src/Count.tsx:1]: do we need react here ?

File: src/components/Card/ProductCard.tsx (hunk 5 above)
Comment (dipeshkumarsah98) [reply src/components/Card/ProductCard.tsx:1]: here as well

**Review by dipeshkumarsah98 — CHANGES_REQUESTED** [review]
resolve new comments

**Pull request merged**